src/xsl/fedora-object-profile.xsl

```diff
@@ -10,17 +10,45 @@
       <xsl:value-of select="local-name(/rdf:RDF/*)"/>
     </xsl:variable>
 
+    <!-- XXX find record created date if there are multiple events -->
+    <xsl:variable name="createdDate">
+      <xsl:choose>
+        <xsl:when test="/rdf:RDF/*/dams:event/dams:DAMSEvent[contains(dams:type,'record created')]">
+          <xsl:value-of select="/rdf:RDF/*/dams:event/dams:DAMSEvent[contains(dams:type,'record created')]/dams:eventDate"/>
+        </xsl:when>
+        <xsl:when test="/rdf:RDF/*/dams:event/dams:DAMSEvent[contains(dams:type,'record edited')]">
+          <xsl:for-each select="/rdf:RDF/*/dams:event/dams:DAMSEvent[contains(dams:type,'record edited')]">
+            <xsl:sort select="dams:eventDate" order="ascending"/>
+            <xsl:if test="position()=1">
+              <xsl:value-of select="dams:eventDate"/>
+            </xsl:if>
+          </xsl:for-each>
+        </xsl:when>
+        <xsl:when test="/rdf:RDF/*/dams:event/dams:DAMSEvent/dams:eventDate">
+          <xsl:value-of select="/rdf:RDF/*/dams:event/dams:DAMSEvent/dams:eventDate"/>
+        </xsl:when>
+        <xsl:otherwise>
+          <xsl:text>1999-12-31T23:59:59-0800</xsl:text>
```

Comment:
Is there a way to have "today" be the fallback instead of this 1999 fixed date? same question for `timestamp`

Hopefully we never actually hit this case. but you never know..

Reply:
@mcritchlow Yes. It shouldn't since we have at lease on event recorded. But if you really want to see the current timestamp here, I think we can update the java codes to past in the current timestamp since I see it trivial to do it in the stylesheet. 
Or we can just ignore it the timstamp in this case (leave it blank) and it's up-to damspas to handle it since the latest date for an old collection will be annoying as well. What do you think?

Reply:
Actually, I think leaving it up to active fedora to manage it is probably the wisest choice. It looks like it will do that if the value doesn't exist. So maybe we should go that route.

Reply:
@mcritchlow I've made the change to remove the fallback and leave it blank, https://github.com/ucsdlib/damsrepo/commits/feature/event_date. But you'd merged the PR while I was pushing my codes. Would you like me to create another PR for it?

Reply:
sorry about that @lsitu  - after considering your response I think leaving it as-is for now seems fine to me. It seems like we have enough new things to test, and it might actually be interesting/easy to do a global query against that date after we're done to see if there are any objects/collections that ended up in that situation.